deps/uv/uv.gyp

```diff
@@ -10,15 +10,19 @@
           ['OS=="solaris"', {
             'cflags': [ '-pthreads' ],
           }],
-          ['OS not in "solaris android"', {
+          ['OS not in "solaris android os390"', {
             'cflags': [ '-pthread' ],
           }],
         ],
       }],
     ],
     'xcode_settings': {
-      'WARNING_CFLAGS': [ '-Wall', '-Wextra', '-Wno-unused-parameter' ],
-      'OTHER_CFLAGS': [ '-g', '--std=gnu89', '-pedantic' ],
+      'conditions':[
+        ['OS!="os390"', {
+          'WARNING_CFLAGS': [ '-Wall', '-Wextra', '-Wno-unused-parameter' ],
+          'OTHER_CFLAGS': [ '-g', '--std=gnu89', '-pedantic' ],
+        }],
+      ]
```

Comment:
Is this change necessary? I would expect `xcode_settings` to only apply to OS X.